### ISSUE TITLE
TypeError: _pusher is not a constructor

### ISSUE THREAD

**Issue by pishguy** [underfisk/nestjs-pusher#20]
````
TypeError: _pusher is not a constructor
    at new PusherService (/usr/src/app/node_modules/.pnpm/nestjs-pusher@2.0.0_@nestjs+common@10.0.5_@nestjs+core@10.0.5_pusher@5.1.3/node_modules/nestjs-pusher/dist/pusher.service.js:129:23)
    at Object.<anonymous> (/usr/src/app/node_modules/.pnpm/nestjs-pusher@2.0.0_@nestjs+common@10.0.5_@nestjs+core@10.0.5_pusher@5.1.3/node_modules/nestjs-pusher/dist/pusher-testing-util.js:14:15)

    at Module._compile (node:internal/modules/cjs/loader:1257:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1311:10)
    at Module.load (node:internal/modules/cjs/loader:1115:32)
    at Function.Module._load (node:internal/modules/cjs/loader:962:12)
    at Module.require (node:internal/modules/cjs/loader:1139:19)
    at require (node:internal/modules/helpers:121:18)
    at Object.<anonymous> (/usr/src/app/node_modules/.pnpm/nestjs-pusher@2.0.0_@nestjs+common@10.0.5_@nestjs+core@10.0.5_pusher@5.1.3/node_modules/nestjs-pusher/dist/index.js:8:14)
    at Module._compile (node:internal/modules/cjs/loader:1257:14)
```

**Comment by pishguy** [comment]
@underfisk not yet but i'm using the latest version of this library

**Comment by pishguy** [comment]
@underfisk  sure, i try to reproduction today. I'll send the `github` repository for you 👍

**Comment by underfisk** [comment]
@pishguy I was playing around with `2.0.0` and i couldn't repro. Also, I've released `2.0.1` (a new patch) that should lazy load the graphql module, and its no longer needs to be installed if you're not using it.
I'm closing to close this issue until a repro is provided but make sure you bump the version and try now on the new patch
